pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/ExpValFunctors.hpp

```diff
@@ -256,6 +256,7 @@ template <class PrecisionT> struct getExpectationValueTwoQubitOpFunctor {
     }
 };
 
+// LCOV_EXCL_START
```

Comment:
We are also excluding this part of the code from coverage. Is there something we can do to include it? Are you planning a follow-up PR?

Reply:
> We are also excluding this part of the code from coverage. Is there something we can do to include it? 

Yes.

> Are you planning a follow-up PR?

But it will be much more straightforward in a follow-up. We are still using the "inner-product" scheme in `expval` instead of `getExpectationValueMultiQubitOpFunctor` on `master`. When we merge #489 , we can simply add an `expval` test to cover it.

Reply:
If by removing the ``LCOV_EXCL`` zone we still can pass the coverage checks, I think it would be preferable.